src/jit/compiler.h

```diff
@@ -6361,6 +6361,23 @@ public :
 
     GenTreePtr                  eeGetPInvokeCookie(CORINFO_SIG_INFO *szMetaSig);
 
+    inline size_t               eeGetPageSize() const
+    {
```

Comment:
This is going to need to read a value from `CORINFO_EE_INFO` soon, at which point they will need to be instance methods.